framework/test/common/utils/random.js

```diff
@@ -86,7 +86,7 @@ random.application = function() {
 		name: random.applicationName(),
 		description: 'Blockchain based home monitoring tool',
 		tags: 'monitoring temperature power sidechain',
-		type: random.number(0, 2),
+		type: 0,
```

Comment:
im not sure why it was random, but should it not be random?

Reply:
`dapp.type` required to be `0` in core and elements logic, therefore tests using this random function will occasionally fail if left `random(0,2)`

Reply:
this part needs to be put it back after the fix in elements.